crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -78,6 +93,156 @@
         java.security.Security.setProperty("networkaddress.cache.ttl", "30");
     }
 
+    @GET
+    @LZF
+    @Path("/searches/{entity}")
```

Comment:
I think I would just keep this as `search` rather than change verbiage.  I think that makes sense.  It shouldn't clash with any of the actual search methods.